### DISCUSSION TITLE
[FR] Add edit link to Field Handles within entries an co.

### DISCUSSION THREAD

**Discussion by davidhellmann** [craftcms/cms#8838] [Ideas]
@brandonkelly related to #5833 it would be nice to have a link to edit the Field directly. I use that very often and is saves a lot of clicks. After edit and save the field a redirect to the entry would be nice too.

**Comment by brandonkelly** [comment]
Kinda related to #2685.

**Comment by mmikkel** [comment]
I'm stoked that the "show field handle" feature is finally in core, but I have to admit that I wish it was implemented more similar to my [CP Field Inspect](https://plugins.craftcms.com/cp-field-inspect) plugin (which currently sits at ~12k active installs). 

In addition to exposing the field handle, CP Field Inspect adds the feature @davidhellmann describes by adding a visible UI element (a little cogwheel button) to the fields, which can be clicked to go to the field's settings page. The plugin also takes you back to the element you were editing, if/when you save the field settings. _This is a massive time-saver when modelling content._

Also, crucially, _copying field handles is easier_ with CP Field Inspect. With the plugin, you can reveal the handle by mousing over the cogwheel, then double-click to select the text and use Cmd+C to copy. Using the core feature, you basically need to hold down the Alt key, then double-click the text to select, then right-click the handle and finally choose copy via the OS context menu (again, all while keeping the Alt key pressed down). TBH I can definitely think of more elegant ways of solving this than using a mouseover in the first place, the point is that being able to easily copy handles is also a big time saver/DX win (more often than not, being able to read the handle is only half the battle; you typically want to bring that handle, without any typos, into a template/config file/whatever).

**Comment by davidhellmann** [comment]
This!

**Comment by piotrpog** [comment]
@mmikkel 
I just wish that Craft devs literally just copied CP field Inspect plugin. It's pretty much perfect :)

**Comment by wbrowar** [comment]
While I realize alt/option are modifier keys for things like mouse clicks, it might work better if you could use them to toggle the state of the field handle. While I got used to this new method pretty quickly, it would be nice to see other admin-specific UI elements hidden until called up. Thinking of things that you may put in the "details" section of the sidebar or maybe even some of the new UI elements introduced in 3.5. It might be out of scope for this issue, but a lot of this could be achieved with a set of CSS classes (like `admin-hide`, `admin-show`) and the use of alt/option to toggle them.

@mmikkel The only issue I really ever have had with CP Field Inspect is a few times where clients saw my screen and asked about the gears above the fields. I’ve gotten into the habit of switching to a different user when showing them around Craft, but it would be nice to have all admin-specific stuff hidden until it's needed (this is certainly subjective).

Finally, the only other thing about this issue that wasn't mentioned above is that in the "details" area of the edit page, CP Field Inspect also included a link back to the field manager for whatever element you’re working in. This is just as handy as the link back to the field setting and it would be great to have it included in Craft, as well.

**Comment by brandonkelly** [comment]
Changed this up for the next release (3.5.3) – see https://github.com/craftcms/cms/issues/6532#issuecomment-671576419.
